python/lsst/ts/vent/controller/dispatcher.py

```diff
@@ -0,0 +1,109 @@
+import logging
+import traceback
+
+from lsst.ts import tcpip
+
+from .controller import Controller
+
+
+def _type_convert(t: type, value: str):
```

Comment:
A couple things here, I think it is usually best to start method names with a verb as they express an action to be taken. Consider renaming this method to `_convert_to_type` or maybe `_cast_string_to_type`, here you are making it explicit that the function is designed to work with string inputs for value.

The other thing is the name of the variable. Usually single letter variables are reserved to loops and such. For a method argument I would suggest you use something more expressive. How about `new_type`?